packages/wallet/.env.example

```diff
@@ -6,4 +7,7 @@ NETWORK_ID=0
 MNEMONIC_WORDS="actor scout worth mansion thumb device mass pave gospel secret height document merge text broom kind lesson invest across estate erase interest end century"
 WALLET_PASSWORD=some_password
 POOL_ID_1=pool1euf2nh92ehqfw7rpd4s9qgq34z8dg4pvfqhjmhggmzk95gcd402
-POOL_ID_2=pool1fghrkl620rl3g54ezv56weeuwlyce2tdannm2hphs62syf3vyyh
\ No newline at end of file
+POOL_ID_2=pool1fghrkl620rl3g54ezv56weeuwlyce2tdannm2hphs62syf3vyyh
+OGMIOS_HOST=localhost
```

Comment:
Since there are now 2 submit providers available, we should probably document the options either as comments in `.env.example` or README.

Reply:
Done in c079f6668a0ede6787ec546c13296a4a04e9ef33 and 2aba96af2329c4a28f078325fe9d8fc7ff3893d4